courseworkA/Markdown report template assignment A .Rmd

```diff
@@ -247,66 +256,234 @@ Describe the statistical test you suggest to care out on the collected data
+abline(v = mean(subset01$pages), col = "green")
+
+d <- density(subset10$pages)
+plot(d, xlab='Number of Page Visits', ylab='Prob. Density', main='Page visits on New version for Consumers entries')
+abline(v = mean(subset10$pages), col = "blue")
+
+d <- density(subset11$pages)
+plot(d, xlab='Number of Page Visits', ylab='Prob. Density', main='Page visits on New version for Company entries')
+abline(v = mean(subset11$pages), col = "orange")
+```
+
+The Density plots indicate that none of the Page visit densities resemble a Gaussian distribtuion, apart from the page visits for New version and Old version company entries. The rest have skewed distributions and the Page visits for Old version consumer entries resembles a mixture of densities. General Linear Model analysis assumes the fact that the target continuous variable has Gaussian-error distribution and thus uses appropriate log-likelihoods for the best MLE regression fit. Since the densities don't resemble Normal distributions, this might hamper the interpretability of the results.
+
+### Frequentist Approach
+
+#### Model analysis
 <!-- Conduct a model analysis, to examine the added values of adding 2 factors and interaction between the factors in the model to predict page visits, and include brief interpretation of the results. -->
 
 
-<!-- ```{r} -->
-<!-- #include your code and output in the document -->
-<!-- ``` -->
+```{r}
+# Model fitting for each factor and a combination of them
+
+model0 <- lm(pages ~ 1, data=data, na.action=na.exclude)
+model1 <- lm(pages ~ version, data=data, na.action=na.exclude)
+model2 <- lm(pages ~ portal, data=data, na.action=na.exclude)
+model3 <- lm(pages ~ version + portal, data=data, na.action=na.exclude)
+model4 <- lm(pages ~ version + portal + version:portal, data=data, na.action=na.exclude)
+
+# ANOVA results of the effect of adding the factors
+
+pander(anova(model0, model1), caption='Version as main effect on Page visits')
+
+pander(anova(model0, model2), caption='Portal as main effect on Page visits')
 
+pander(anova(model3, model4), caption='Interaction effect vs 2 main effects')
 
-<!-- #### Simple effect analysis -->
+pander(anova(model4), caption='Version, Portal and interaction effect on Page visits')
+
+# AICc scores of the models
+
+models <-list(model0, model1, model2, model3, model4)
+model.names <-c("model0","model1","model2","model3","model4")
+pander(aictab(cand.set = models, modnames=model.names))
+```
+
+The ANOVA results for the comparison of each model type indicate that the added values by including the factors individually, together and their interaction effect is statistically significant since all their p-values are <0.001. Out of all these, the added value is the most significant in the case of just adding the factor of Portal since it's p-value is the least (1.28e-37). The AICc results show that model4 has the best goodness of fit since its corrected-AIC value is the least with the best log-likelihood score too.
```

Comment:
instead of saying the p-value is "the least" I'd say the "lowest" or "smallest". Same for AIC value